src/java/fr/paris/lutece/plugins/suggest/utils/ProcessBBCode.java

```diff
@@ -44,22 +44,6 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-/*
```

Comment:
Are you sure your are allowed to relicense this code ? This is not a code improvement, and should really by (if the change is appropriate) a separate commit with explicit comments about the license change

Reply:
Actually, no. I'll restore this.